Example/Example/EditorDemoController.swift

```diff
@@ -1032,8 +1032,32 @@ extension EditorDemoController {
 
 extension EditorDemoController: TextViewAttachmentDelegate {
 
-    func textView(_ textView: TextView, attachment: NSTextAttachment, imageAt url: URL, onSuccess success: @escaping (UIImage) -> Void, onFailure failure: @escaping () -> Void) {
+    func exportPreviewImageForVideo(atURL url: URL, onCompletion: @escaping (UIImage) -> (), onError: @escaping () -> ()) {
```

Comment:
Let's move this to a more appropriate location since it's really not related to `TextViewAttachmentDelegate`.